HelpSource/Classes/Event.schelp

```diff
@@ -36,31 +36,91 @@ Returns an empty event with link::#.defaultParentEvent:: as parent.
 method::silent
 Returns an event that describes a pause of strong::dur:: duration.
 
+
+
 method::addEventType
-Event types define alternate play functions that are selected by the value of strong::~type::.
+Event types define alternate play functions and parent events that are selected by the value of strong::~type::.
+
+
 argument::type
 A name (usually a symbol) for the event type, which can be used to select it
 
 argument::func
 A function which optionally takes the server as a first argument
 
+argument::parentEvent
+An event with default values which is used to override the parent event. If code::parentEvent:: in turn has no parent, its default parent event is set.
+
+discussion::
 code::
-Event.addEventType(\happyEvent, { ("I am so happy to be silent sometimes, says" + ~who).postln; });
+Event.addEventType(\happyEvent, { "I am so happy to be silent sometimes, says %.\n".postf(~who) });
+(type: \happyEvent, who: "Alice").play;
+
+// using protoEvent
+(
+Event.addEventType(\happyEvent, {
+	"I am so happy to be silent sometimes, says %.\n".postf(~who)
+}, (who: "Alice"));
+)
+
+(type: \happyEvent).play; // use default
+(type: \happyEvent, who: "Eve").play; // overrride default
+
+// in a Pbind:
 Pbind(\type, \happyEvent, \who, Prand(["Alice", "Bob", "Eve"], inf), \dur, Pwhite(0.1, 1.0, inf)).play;
 
-// To a certain degree, it is possible to reuse some of another event type's functionality:
+// protoEvent for indirect calls:
```

Comment:
at this point, protoEvent is not mentioned in the document, and here it is used implicitly. Makes it hard to follow.